src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -353,6 +373,13 @@ void drawing_classifier::init_training(
   // by the data iterator.
   nn_spec_ = init_model();
 
+  if ( enable_warmstart ) {
+    // Initialize the neural net with warm start model weights.
+    model_spec warmstart_model(mlmodel_path);
+    float_array_map trained_weights = warmstart_model.export_params_view();
```

Comment:
the 6.0 layer name might be different from the warm_start model.

Reply:
different how?

Reply:
Also specifically what change are you requesting here?

Reply:
the layer name before has suffix `_fwd` but this release doesn't have. You may check the implementation of update_weights.

Reply:
the coreml model does not have the suffix `fwd`

Reply:
I manually removed it and uploaded the model again. A number of cleanups were done on the CoreML model (pasting from my previous standup in case you missed):

> I uploaded the Drawing Classifier CoreML model with the valid author and license metadata and the license txt file is uploaded and CoreML model now contains only the relevant layers (i.e. removed dense1, softmax, preprocessing etc), edited model output field to match dense0 output (removing ClassProbabilites and ClassLabels) and converted from neuralNetworkClassifier to neuralNetwork since it is no longer a classification network. Added the supporting code to fetch the MLmodel path from Python and load in C++. Added warm_start as a user option in C++. My focus is to ensure the warm start model correctly reaches the Python TF train(). Removed the additional fwd suffix in the CoreML model layer names which is not expected by the TF implementation of DC.

Reply:
Good. That's good enough for this release. Let's do this pragmatically next time.

Reply:
What wasn't pragmatic about this?

Reply:
I don't think I follow what you mean to say -  what isn't good enough? and what can be done more pragmatically next time?

Reply:
Sorry I didn't make myself clearer. Think about you have more than 10 warm start models to try on, and you don't want to manually do it for each of them (even though, you may want to have a pipeline to process them all). You can have a function to match the warm start layer name with the current layer name format in DC. This will loosely decouple your dependency on the **exact** naming format. If we want to add the `_fwd` suffix back or add some extra info, we don't need to modify the warm start model and upload the modified copy again. A potential solution is to use `regex`.

Reply:
Yeah that's a great point! Toby shared a script with me that helped me do a lot of the processing on the CoreML model, so we can set up a script to do the CoreML model pre-processing for DC warmstart models. The idea was also to remove any additional meta data or layers that we do not need and make the uploaded model file as compact as possible.

For name matching in the warm start model and the layer name, we could introduce name cleanup in the code. My options to deal with the suffix was either do it in code or do it as part of the CoreML model cleanup so I went with the latter because I didn't want to expose a user with layer names that were not consistent with the layer names we define in TF implementation. Ideally if a user opens the warmstart model in coreml, they should see the same layers names as ones we use in our code.

Reply:
excellent!